src/Resources/Locales/cs.json

```diff
@@ -1,915 +1,917 @@
 {
-    "languageName": "Czech",
```

Comment:
It looks like the change has altered all of the tab indents for these files? Was this intentional?

Reply:
@jamescarpinter I believe it now has the same indent as the other language files (except english)